### ISSUE TITLE
AutoComplete: Shift key not working for selecting

### ISSUE THREAD

**Issue by forNoor** [primefaces/primevue#2930]
### Describe the bug

when we are using input text, we need to select currnet text. one way to select is using shift+home or shift+end. (for selecting from start to current or from current to end). this feature is not possible in autocomplete component.

### PrimeVue version

3.16.2

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

**Comment by forNoor** [comment]
this is bug not enhacement. because all of input elements support this.
i think you handle keyboard in autocomplete component and beacause of that, this problem occured.

**Comment by titou10titou10** [comment]
I agree with @forNoor , it is a bug... Not being able to perform a SHIFT+END then DELETE to clear the value present in the box is very very annoying
Other components allow this as input text box all over internet
